lib/mvgApi.js

```diff
@@ -2,7 +2,7 @@
 const cheerio = require("cheerio");
 const request = require("request");
 const Line = require("./Line");
-const iconv = require("iconv");
+const iconv = require("iconv-lite");
 
```

Comment:
Add a const for the default value here
`const defaultApi = "https://www.mvg-live.de/ims/dfiStaticAuswahl.svc";`